GameServer/Logic/Packets/Packets.cs

```diff
@@ -3214,6 +3348,16 @@ public SetCooldown(uint netId, byte slotId, float currentCd, float totalCd = 0.0
         }
     }
 
+    public class SetItemStacks : BasePacket
+    {
+        public SetItemStacks(Unit unit, byte slot, byte stack1, byte stack2) : base(PacketCmdS2C.PKT_S2C_SetItemStacks, unit.getNetId())
```

Comment:
120 character limit please :)

I'd suggest you to add a marker at the 120 character mark in your editor. In case you use VS, the Editor Guidelines plugin should allow you to do so (linked in readme)